seq2science/cli.py

```diff
@@ -287,8 +293,51 @@ def _run(args, base_dir, workflows_dir, config_path):
     parsed_args["config"].update({"cores": parsed_args["cores"]})
     resource_parser(parsed_args)
 
-    # run snakemake
+    # run snakemake/seq2science
+    #   1. pretty welcome message
+    setup_logger()
+    log_welcome(logger, parsed_args)
+    if not args.skip_rerun or args.unlock:
```

Comment:
it would be nice if `--skip-rerun` is on when you haven't started a run yet. 
This could be accomplished with `if os.path.exists(config["results_dir"]) and (not args.skip_rerun or args.unlock):`

buuuut this may be too tedious for a one-time benefit! I just realized we dont have access to `config` yet.